samples/matter/common/src/app/fabric_table_delegate.h

```diff
@@ -57,10 +58,18 @@ class AppFabricTableDelegate : public chip::FabricTable::Delegate {
 #if CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT
 				chip::DeviceLayer::ThreadStackMgr().ClearAllSrpHostAndServices();
 #endif // CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT
+#if !defined(CONFIG_SOC_SERIES_NRF53X) && !defined(CONFIG_SOC_SERIES_NRF54HX)
```

Comment:
Let's try to use maybe `CONFIG_MPSL` instead of listing the specific socs.

Reply:
Added `CONFIG_SOC_FLASH_NRF_RADIO_SYNC_MPSL` which is more accurate for this purpose.